pkg/products/monitoring/prometheusRules.go

```diff
@@ -292,6 +292,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "install-upgrade-alerts",
```

Comment:
Is this an appropriate name? or perhaps this should be under a different, already existing alert?

Reply:
This seems ok